pkg/orchestrator/pod.go

```diff
@@ -28,13 +28,12 @@ import (
 )
 
 const (
-	redactedValue = "********"
 	// from https://github.com/kubernetes/kubernetes/blob/abe6321296123aaba8e83978f7d17951ab1b64fd/pkg/util/node/node.go#L43
 	nodeUnreachablePodReason = "NodeLost"
 )
 
 // ProcessPodList processes a pod list into process messages
-func ProcessPodList(podList []*v1.Pod, groupID int32, hostName string, clusterName string, clusterID string, withScrubbing bool, maxPerMessage int, scrubber *DataScrubber, extraTags []string) ([]model.MessageBody, error) {
+func ProcessPodList(podList []*v1.Pod, groupID int32, hostName string, clusterID string, config *config.OrchestratorConfig, extraTags []string) ([]model.MessageBody, error) {
```

Comment:
nit: is there a reason you call this here config and in the other processer (DCA) places cfg? Is that on purpose?